### PR TITLE
Move more line break data into CPT

### PR DESCRIPTION
**Pull request by sffc**
This is an *optional* pull request. I would like feedback from the code owners (@makotokato, @aethanyc) on whether this is a positive change.

### CODE REVIEW

File: provider/datagen/src/transform/segmenter/mod.rs [hunk 1]
```diff
@@ -333,8 +333,9 @@ impl SegmenterRuleProvider {
         // The table length should be large enough to contain all codepoints.
         const UAX29_CODEPOINT_TABLE_LEN: usize = 0xE1000;
 
-        // The property values of codepoints >= U+0x20000 are built into the line segmenter.
-        const UAX14_CODEPOINT_TABLE_LEN: usize = 0x20000;
+        // The property values of codepoints >= U+0x20000 could be built into the line segmenter,
+        // but we instead store them compactly in the CodePointTrie.
+        const UAX14_CODEPOINT_TABLE_LEN: usize = 0xE1000;
```
Comment (makotokato) [inline provider/datagen/src/transform/segmenter/mod.rs:338]: UAX29's table size and UAX14 table size are same now, so it is unnecessary to define two value.

**Review by makotokato — APPROVED** [review]

**Review by aethanyc — APPROVED** [review]
Nice! These codepoints > 0x20000 are already defined in Line_Break property.

**Comment by aethanyc** [comment]
We have a conflicts in `testdata.postcard`, which needs to be fixed before merging.

**Comment by aethanyc** [comment]
@sffc I take a stab at you PR to rebase it and address Makoto's comment, but I cannot push to your branch. Do you mind taking the branch from https://github.com/aethanyc/icu4x/commits/simplify-line, and merge the PR? (assuming the `CODEPOINT_TABLE_LEN` commit looks good to you)

**Comment by sffc** [comment]
@aethanyc Can you open a new PR from your branch and we can merge that one and close this one?

**Comment by aethanyc** [comment]
Sure. Open #2165, and close this.

**Pull request closed without merging**